runtime/autoload/man.vim

```diff
@@ -67,14 +67,51 @@ function! man#read_page(ref) abort
   call s:read_page(path)
 endfunction
 
+" Handler for s:system() function.
+function! s:system_handler(jobid, data, event) abort
+  if a:event == 'stdout' || a:event == 'stderr'
```

Comment:
we want to ignore stderr for man.vim, right? So that should be removed from the condition